### ISSUE TITLE
Migaku official Chrome extension creates empty cards from Reader

### ISSUE THREAD

**Issue by heberleh** [migaku-official/migaku-browser-extension-issue-tracker#31]
**Describe the bug**
If you open a text using MigakuReader (or anywhere else I think), parse the text, and click export... it doesn't work.
It opens the dictionary, but the Default deck of Anki is full of empty cards.

I clicked export all cards because I'm doing a course and I want to practice literally all words from the texts.

**Screenshots**
![image](https://user-images.githubusercontent.com/4249107/156923460-d089521b-710e-4897-afc6-3075327dba16.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 0.9.9.2

**Comment by KieranBrannigan** [comment]
Hi, could you check that your Migaku Dictionary exporter is set up correctly?
 
Please note: We recommend using Migaku note types built-in to the Migaku Anki Addon for the easiest set up.

Here is a video explaining how to set it up:
https://www.migaku.io/video-guides/the_card_exporter

Also note the tags feature, so you could add tags for the exam you are studying for so you can find these sentences again easily in the anki browser ;)

**Comment by heberleh** [comment]
Thanks for the fast answer, @KieranMigaku 
Indeed all the settings were not echoed from the Dictionary settings from the main settings... Like the Language, or which dictionaries are activated, or the order of dictionaries.

Then I opened the dictionary, and it's local settings... and there it was all empty. I set it up... 
Now it was indeed processing, taking time to create cards. However, they appear still all empty.
Something is still missing.

![image](https://user-images.githubusercontent.com/4249107/156924370-ef70953f-3f8b-48bb-9bb7-234ce4b7c1b4.png)

**Comment by KieranBrannigan** [comment]
So the cards are being added but all the fields are empty, or just the definitions are empty?

If just the definitions are empty you can try this:
<img width="476" alt="image" src="https://user-images.githubusercontent.com/60760551/156924434-ac988d9c-ed58-4b5c-8b55-a4a2632aff5b.png">
Press this dropdown button here, and make sure the selected language is correct (german), and try searching some words to make sure the dictionary is working.

Also can you double check what version of the Migaku Anki Addon you are using?

**Comment by heberleh** [comment]
It's German, it's there
![image](https://user-images.githubusercontent.com/4249107/156924936-becf70af-2bc6-433e-8c57-f85fba392118.png)

Anki extension 0.1.0-alpha.17

**Comment by heberleh** [comment]
However, I noticed that this new extension forced me somehow to update Anki.
Before, I was using the recommended version.
The new instructions didn't mention any Anki version.
![image](https://user-images.githubusercontent.com/4249107/156925138-3162ff15-7db5-410b-87ea-ea8bbb8a704d.png)

**Comment by KieranBrannigan** [comment]
> However, I noticed that this new extension forced me somehow to update Anki. Before, I was using the recommended version. The new instructions didn't mention any Anki version.

What version of Anki are you on now? It should be 2.1.49

Also, try exporting one card to the exporter with "Add cards automatically" unchecked and check if the sentence is added to the card exporter and check the dropdowns are selected appropriately:
<img width="374" alt="image" src="https://user-images.githubusercontent.com/60760551/156925197-90702a4b-bf0f-4d62-99b6-72aa6eda416b.png">
 
![image](https://user-images.githubusercontent.com/60760551/156925217-2002e8d4-56e4-4200-8ea5-543fbec93dfa.png)

Sorry if you have already done this but just need to make sure

**Comment by heberleh** [comment]
That's it. I overlooked this part. 

Before, all of these were predefined. Now I need to manually match all of them.

However I can't find a "Target Word" option to select, only "target wordS". I mean, there is only the "Unknown WordS" option.

![image](https://user-images.githubusercontent.com/4249107/156925773-941f046b-7268-497c-bcfa-8d2a0e6fbe0a.png)
Any possibility to do as before? Each of these words should be in a different row in Anki browser (card).

I would like to have each of this word as a Word Audio Card.

I couldn't find a way to add them while reading the text (automatically). The Ctrl + Q option seems to behave like this version.
Thanks!

**Comment by KieranBrannigan** [comment]
> Before, all of these were predefined. Now I need to manually match all of them.

This means you have probably accidentally selected the wrong note type, or an old Migaku Note type. To get the correct one you need to check german from `Migaku -> Settings/Help -> Language Selection` and select German:
![image](https://user-images.githubusercontent.com/60760551/156925945-25d53dad-21cf-414f-9bfb-dfdec050e91d.png)
![image](https://user-images.githubusercontent.com/60760551/156925948-7dfa6f9d-69f0-42f5-a0ff-479ef88973be.png)


This will add the correct note type which you can use. When you select this note type you will see these 4 boxes at the top of the card creator: 
<img width="390" alt="image" src="https://user-images.githubusercontent.com/60760551/156925979-f8080750-42a1-4e02-9345-7086fa72d49e.png">




> Any possibility to do as before? Each of these words should be in a different row in Anki browser (card).
> 
> I would like to have each of this word as a Word Audio Card.

Unfortunately I do not believe this is possible. The Migaku Browser Extension will export All Sentences, but will only create one card per sentence.

**Comment by heberleh** [comment]
This was already selected, but I don't see the same as you see
![image](https://user-images.githubusercontent.com/4249107/156929545-70884ee0-3a1a-47e2-8923-227686d849c3.png)

**Comment by heberleh** [comment]
Ah... I found it!
There is one option that is only Migaku German, then it appears.
At the same time I can't remove the old ones, I guess

**Comment by heberleh** [comment]
Yeah, I don't know what's going on. I'm trying... 
![image](https://user-images.githubusercontent.com/4249107/156930107-475b8a2d-a67b-44ac-84a4-f31d31058f5e.png)

**Comment by KieranBrannigan** [comment]
Awesome! 

Btw you can remove the old note types in Anki, however make sure you convert them all before you delete the note type, or you will lose any cards using that note type.

**Comment by KieranBrannigan** [comment]
Also note after making changes in Anki you will need to re-open the Migaku Dictionary (or refresh with `f5`)

**Comment by heberleh** [comment]
Refreshing the reader restored to 26 cards instead of 72 broken cards.
I'm still trying to get them to work.
It's also not respecting the order of the Fields, nor the maximum definitions (e.g., I see 6 images instead of 3). 

Too many hours  to set it up. I may give up and use Readlang again. I'm closing the issue, because I changed the topic. Original issue was solved.

![image](https://user-images.githubusercontent.com/4249107/156930198-d383488d-8413-4804-9bb8-b78c496af9d1.png)

![image](https://user-images.githubusercontent.com/4249107/156930264-1a075ac0-d4e1-4378-8946-dde138efa9d9.png)

![image](https://user-images.githubusercontent.com/4249107/156930543-5a030dee-5b4c-4506-91ba-a6499ec9658f.png)
